recipes/acquia/builder.js

```diff
@@ -29,7 +29,6 @@ module.exports = {
       // Load .env file.
       options.env_file = ['.env'];
       options.webroot = 'docroot';
-      options.xdebug = false;
```

Comment:
@cdubz note that this won't override what the user puts in their Landofile, it just sets the default. Guessing the other issues people are reporting on https://github.com/lando/acquia/issues/6 are xdebug setup being finicky.

I don't think it makes sense to remove the default disabling of xdebug, since that's the pattern other Lando recipes follow. @pirog I'm guessing xdebug needs to be a non-null value?

Reply:
@reynoldsalec If I add `xdebug: true` to a `.lando.yml` file with the `drupal9` recipe everything works as expected. Changing to the `acquia` recipe it just does not work at all (rebuilds included). Back when I made this PR I tested this pretty extensively and it seems pretty clear that this was the issue.

https://github.com/lando/acquia/blob/245a55fbd2f39df9806bfe7b51337d9f5e435930/recipes/acquia/builder.js#L27 merges the defaults and overrides so the fact that the `xdebug`, `webroot`, and `env_file` directives come _after_ that means any `.lando.yml` overrides are overwritten, right? Or am I missing something?

Even if I'm missing something, the parent [`_drupaly`](https://github.com/lando/cli/blob/main/plugins/lando-recipes/types/drupaly/builder.js) appears to set defaults in the `config` array (as opposed to the constructor) so it still seems like that would make more sense. Moreover, the `xdebug` default is already `false` so it doesn't seem to make sense to set it in the `acquia` recipe as well.

Reply:
I also took a look at some other recipes to compare. Any recipe dealing with options in the constructor seems to do it with merge operations (as opposed to setting the values explicitly). The one example of that I did find is in the Pantheon recipe and that is documented as being done specifically to enforce a setting: https://github.com/lando/pantheon/blob/main/recipes/pantheon/builder.js#L118 (i.e., not allow the user to override it).

Reply:
You're totally right, I misread the Acquia builder file, don't know why that's specified. Tested with your patch and that installs xdebug correctly when I specify `xdebug: true` in my .lando.yml. 

Agreed it is a little strange that Lando has `xdebug: false` defined at multiple levels of inheritance. Could simply be copy-and-pasting to create the services.

Reply:
Ok phew I’m not crazy 😜

Reply:
Ha, pretty sure it's just me being dense, kept thinking you were talking about the config array options and wasn't reading the diff correctly, sorry about that! 🤦‍♂️